### PR TITLE
(#3381) Update Blogs for NCIDS

### LINKED ISSUE #3381: Story: Update Blog Series for NCIDS Layout
## As a user, I would like to view Blog Series content using the NCIDS, so that I have a better navigation experience.

This ticket is to update blog series templates that display the layout with the content on the left and the blog featured posts/categories/archive on the right.

As part of this ticket, the Inner Pages background color should be updated to be #FFFFFF

> **ESTIMATE** 8

### Acceptance Criteria

```gherkin
Scenario: [title]
  Given [context]
    And [some more context]
  When  [event]
  Then  [outcome]
    And [another outcome]
```

### Resources:
* Feature: JIRA Ticket #
* Mockups: *Here goes a URL to or the name of the mockup(s) in inVision*
* Web Page: *Here goes a URL to an existing web page*

### Notes

*Some complementary notes if necessary:*

* > Here goes a quote from an email
* Here goes whatever useful information can exist…

---

### Prerequisites
- *required dependency for identifying the problem, scope and then solution for this issue*
- [x] #3240
- [x] #3375 

### Tasks
- [ ] Setup blog post and series entrypoint
   - You will need to update the ncids_trans.libraries.yaml to change the "entry points" to have a dependency on ncids-global instead of the legacy-global.
- [ ] Update to match the Spike template
  - `templates/content/blog/page--cgov-blog-post.html.twig`
  - `templates/content/blog/page--cgov-blog-series.html.twig`
- [ ] Update the node templates (listed below) to:
   - swap out old container divs with new ones
   - update the main-content ids (so the skipnav works) 
   - add the legacy wrapper
   - remove the page options. This should be removing `{{ drupal_block('page_options') }}` in most cases.
- [ ] fix issue with getting Page Title on breadcrumbs https://github.com/NCIOCPL/cgov-digital-platform/issues/3366
- [ ] What to do with the right sidebar (Feature posts, categories, archive)? Does it make more sense to redo the component or just the existing legacy stuff

#### Node templates
- [ ] `templates/content/blog/node--cgov-blog-post--full.html.twig`
  - https://www.cancer.gov/news-events/cancer-currents-blog/2022/ivosidenib-chemotherapy-aml-idh1
- [ ] `templates/content/blog/node--cgov-blog-series--full.html.twig`
  - https://www.cancer.gov/news-events/cancer-currents-blog
  - https://www.cancer.gov/about-nci/organization/ccg/blog
  - https://www.cancer.gov/grants-training/nci-bottom-line-blog
  - DCEG: https://dceg.cancer.gov/news-events/news
  - NCI Connect: https://www.cancer.gov/rare-brain-spine-tumor/blog
  - 

### Technical Notes
- Series template: https://www.cancer.gov/news-events/cancer-currents-blog, https://www.cancer.gov/news-events/cancer-currents-blog?topic=prevention
- Post template: https://www.cancer.gov/news-events/cancer-currents-blog/2022/nivolumab-chemotherapy-neoadjuvant-lung-cancer
- We need to work on on entrypoints for blog posts and blog series, it doesn't make sense to have the same Common.scss

### PR DESCRIPTION
**Pull request by bryanpnih**
ODE: https://ncigovcdode476.prod.acquia-sites.com/
Closes #3381

### CODE REVIEW

**Comment by mworrest** [comment]
1. Bullets for archive in CCG blog are going outside of the box. https://ncigovcdode476.prod.acquia-sites.com/about-nci/organization/ccg/blo
Screen Shot 2022-07-21 at 11 58 13 AM
g

**Comment by meglake** [comment]
@bryanpizzillo, adding issues at they come up to Michelle's comment above so we have a single numbered list of issues. Let me know if you have questions!

**Comment by bryanpizzillo** [comment]
@meglake @mworrest 1&2 are fixed.

**Comment by bryanpizzillo** [comment]
@meglake @mworrest 3&4 are fixed on the ode now.

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/legacy/styles/_blogs.scss [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+/*Begin Blog Content CSS*/
+.blog-post-publishing-info {
+	font-size: 12px;
+}
+p.blog-post-publishing-info {
+	margin-top: 0;
+}
+.blog.body-banner:before {
+	@include call-to-action-corner();
+	content: "Blog";
```
Comment (olitharp-nci) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/legacy/styles/_blogs.scss:70]: Does this need to be translated?

**Review by olitharp-nci — APPROVED** [review]

**Comment by meglake** [comment]
@bryanpizzillo, regarding #3 - this is now fixed however when the 'Archive' is expanded it shows the years expanded as well without any option to close them (https://ncigovcdode476.prod.acquia-sites.com/news-events/cancer-currents-blog/2021/cervical-cancer-minimally-invasive-surgery-declining). Is there a way to make the years accordions as well like on the live site (https://www.cancer.gov/news-events/cancer-currents-blog/2021/cervical-cancer-minimally-invasive-surgery-declining)

**Comment by meglake** [comment]
@bryanpizzillo blogs have passed client testing. This PR can be approved.

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**